etna/transforms/math/statistics.py

```diff
@@ -445,6 +445,59 @@ def _aggregate_window(self, series: pd.Series) -> float:
         return tmp_series.median(**self.kwargs)
 
 
+class MADTransform(WindowStatisticsTransform):
+    """MADTransform computes Mean Absolute Deviation over the window."""
+
+    def __init__(
+        self,
+        in_column: str,
+        window: int,
+        seasonality: int = 1,
+        min_periods: int = 1,
+        fillna: float = 0,
+        out_column: Optional[str] = None,
+    ):
+        """Init MADTransform.
+
+        Parameters
+        ----------
+        in_column: str
+            name of processed column
+        window: int
+            size of window to aggregate
+        out_column: str, optional
+            result column name. If not given use __repr__()
+        seasonality: int
+            seasonality of lags to compute window's aggregation with
+        min_periods: int
+            min number of targets in window to compute aggregation; if there is less than min_periods number of targets
+            return None
+        fillna: float
+            value to fill results NaNs with
+        """
+        self.in_column = in_column
+        self.window = window
+        self.seasonality = seasonality
+        self.min_periods = min_periods
+        self.fillna = fillna
+        self.out_column = out_column
+        super().__init__(
```

Comment:
Better to follow the order of the parameters of the superclass

Reply:
Arguments orders in `WindowStatisticsTransform` and all it's inheritors are different. 
I think it's better to make `MADTransform`'s argument order the same with `MeanTransform`, `MaxTransform`, etc. 
Moreover, I think that we should change the arguments order in `WindowStatisticsTransform` but not at this PR (looks like big change).